Source/WebKit/UIProcess/API/Cocoa/WKWebpagePreferences.mm

```diff
@@ -586,14 +586,31 @@ - (void)setLockdownModeEnabled:(BOOL)lockdownModeEnabled
 #endif
 }
 
-- (BOOL)_networkConnectionIntegrityEnabled
+- (_WKWebsiteNetworkConnectionIntegrityPolicy)_networkConnectionIntegrityPolicy
 {
-    return _websitePolicies->networkConnectionIntegrityEnabled();
+    _WKWebsiteNetworkConnectionIntegrityPolicy nCI = 0;
+    auto nCIOptions = _websitePolicies->networkConnectionIntegrityPolicy();
```

Comment:
Nit - per our [style guidelines](https://webkit.org/code-style-guidelines/#names), this should be `networkConnectionIntegrityPolicy` (or, simply `policy` / `webCorePolicy` should be fine too).